common/gsLargeInt.c

```diff
@@ -1498,7 +1507,7 @@ gsi_bool gsiLargeIntMultM(gsLargeInt_t *x, gsLargeInt_t *y, const gsLargeInt_t *
 	{
 		if (gsi_is_false(gsiLargeIntSub(m->mData, m->mLength, &temp[logB_r], tempLen - logB_r, dest->mData, &dest->mLength)))
 		{
-			memset(temp, 0, sizeof(temp));
+			SafeZeroMemory(temp, sizeof(temp));
```

Comment:
I would change the function name to gsiZeroMemory and move it's delcaration elsewhere (not in gsLargeInt) so other files could use it. common\gsMemory.c/h would be a good place

Reply:
https://github.com/GameProgressive/UniSpySDK/pull/27/commits/9c1ad7bf869cd1e6da8fb92c464b8176264ccfc4